site/en/docs/extensions/reference/declarativeContent/index.md

```diff
@@ -2,64 +2,79 @@
 api: declarativeContent
 ---
 
+{% Aside %}
+
+To transition from page action to action, see [Emulating pageActions with
+declarativeContent][emulating-page-actions]
+
+{% endAside %}
+
 ## Usage
 
-The Declarative Content API allows you to show your extension's [page action][1] depending on the
-URL of a web page and the CSS selectors its content matches, without needing to take a [host
-permission][2] or inject a [content script][3]. Use the [activeTab][4] permission in order to be
-able to interact with a page after the user clicks on your page action.
+{% Aside 'key-term' %}
+
+An [_extension's action_][api-action] controls the extension's toolbar icon.
+
+{% endAside %}
+
+The Declarative Content API allows you to enable your extension's action depending on the URL of a
+web page and the CSS selectors, without needing to add [host permissions][docs-host-perm] or inject a
```

Comment:
>and the CSS selectors.

What is this telling me? I don't understand this part.

Reply:
I just made a change, let me know what you think